test/code_fund/campaigns/campaigns_test.exs

```diff
@@ -1,10 +1,37 @@
-# defmodule CodeFund.CampaignsTest do
-#   use CodeFund.DataCase
+defmodule CodeFund.CampaignsTest do
+  use CodeFund.DataCase
+  import CodeFund.Factory
 
-#   alias CodeFund.Campaigns
+  alias CodeFund.Campaigns
 
-#   describe "campaigns" do
-#     alias CodeFund.Schema.Campaign
+  describe "campaigns" do
+    test "get_active_campaign_for_property_with_biggest_bid_amount/1 returns the campaign for a property with largest bid amount" do
+      property = insert(:property)
+
+      campaign = insert(:campaign, bid_amount: Decimal.new(10.00))
+      insert(:sponsorship, campaign: campaign, property: property)
+
+      insert(:campaign, bid_amount: Decimal.new(1.00))
+      insert(:sponsorship, campaign: campaign, property: property)
+
+      assert Campaigns.get_active_campaign_for_property_with_biggest_bid_amount(property).id ==
+               campaign.id
+    end
+
+    test "get_random_active_campaign_for_property/1 returns a random campaign for the property" do
+      property = insert(:property)
+      insert(:sponsorship, property: property)
```

Comment:
maybe modify this test to set the bid price on the sponsorships to the same and a 3rd with a lower, then another test that has 2 matching and one higher. Does this make sense?